### ISSUE TITLE
Following() returns empty result when used on user other than the logged in one

### ISSUE THREAD

**Issue by buckket** [ahmdrz/goinsta#305]
I’m trying to get followings of a specific user, but the result is always empty.

After debugging the problem I found out that the API endpoint Following() requests returns an integer in the `next_max_id` field, whilst goinsta expects it to be a string, which it still is on other API endpoints like Followers() but not in Following().

It evens seems to change depending on which user the request in acting on, hence the original problem of this issue.

**Comment by falzm** [comment]
Actually the followings are also empty for the logged user. I'd like to lookup the logged user's followings in order to have a more dynamic users list in [instafeed](https://github.com/falzm/instafeed), but this issue is preventing me to do so:

```console
$ go run instafeed.go
insta.Account.Following().Users=[]goinsta.User(nil)
# @Instagram: 19 followings
```

@ahmdrz can you please have a look at this issue?

**Comment by buckket** [comment]
@falzm I fixed the issue in PR #306, while we’re waiting to get this merged you can use my branch of goinsta or cherry-pick the changeset (it’s nothing fancy).
